docs/conf/eu/2017/unconference.rst

```diff
@@ -0,0 +1,69 @@
+:template: 2017/eu-content.html
+
+Unconference
+============
+
+As part of last year's conference, we held an
+`unconference <http://en.wikipedia.org/wiki/Unconference>`__ across the main stage room.
```

Comment:
Should possibly be a single underscore? But looks like it works so it is probably correct.